reana_server/rest/ping.py

```diff
@@ -10,7 +10,7 @@
 
 from flask import Blueprint, jsonify
 
-blueprint = Blueprint('ping', __name__)
+blueprint = Blueprint('ping', __name__, url_prefix='/reana-api')
```

Comment:
We can't use `/api` because of Invenio's `/api`? If that's the case, we will have to fix the auth on the API application since this is going to break `reana-client`'s calls.